interfaces/issue-data.ts

```diff
@@ -55,6 +56,11 @@ export interface IssueData {
   benefactors?: fundingBenefactor[];
   disputes?: Disputes[];
   payments: Payment[];
+  network?: {
+    colors: ThemeColors;
```

Comment:
_we don't need a complete copy of the theme colours, right?_ we just need the primary?